### ISSUE TITLE
Can't use with 1.31 version

### ISSUE THREAD

**Issue by wuliriki** [davidsantos-br/ETS2Sync-Helper-4#31]
Can't use with 1.31  version

**Comment by Michl7** [comment]
If you have the Scandinavia DLC. Dont check the Checkbox for it.

**Comment by DaviMedrade** [comment]
I updated the job lists but for some reason syncing with the Scandinavia DLC selected still causes an error. I'm working on it, but until that is fixed you can sync as long as you don't select Scandinavia.

**Comment by DaviMedrade** [comment]
This should work with the current version of the app (the changes were made on the server). There is no update to download.

**Comment by DaviMedrade** [comment]
Fixed.

**Comment by skynet2** [comment]
Hi @davidsantos-br ,

Looks like it was not fixed. After sync with all my DLCs + Scandinavia DLC i was transferred to my home city. 
Sync without Scandinavia DLC works well.

Thanks,
Stas

**Comment by maxinnee** [comment]
same problem me too, please fix. i need so badly this sync.

**Comment by maxinnee** [comment]
still not fixed

**Comment by maxinnee** [comment]
waiting for more 6 days to response*

**Comment by DaviMedrade** [comment]
I'm so sorry. It was an oversight on my part.

I have two setups for retrieving the job lists from the server. One is the production setup, used by everyone, and the other is for testing (so I don't disrupt the real one when I'm figuring out a problem or testing a fix).

I was using the testing setup to fix this problem. Once it was fixed, I tested it thoroughly and didn't have any problems. But then I forgot to copy the correction from the testing setup to the production setup (the one used by everyone). That's why when I tested it it worked, but when everyone else tried it, it still failed.

This has been fixed now.
